developer/src/tike/xml/layoutbuilder/builder.xsl

```diff
@@ -247,6 +247,10 @@
           <label for='selSubKeyNextLayer'>Next Layer:</label>
           <select id='selSubKeyNextLayer'></select>
         </div>
+        <div class='toolbar-item'>
+          <label for='chkSubKeyIsDefault'>Is default:</label>
```

Comment:
Is the UX clear what "Is default" does? Possible confusion in the editor if "Key Type" dropdown has "Default".

"Default longpress key" might be too long, but perhaps something similar?

Reply:
Perhaps "Auto-select"?